content/sensu-go/6.3/operations/deploy-sensu/generate-certificates.md

```diff
@@ -290,12 +290,125 @@ Reopen the root CA certificate, expand Trust, select Always Trust, and save your
 Delete the root CA certificate from login.
 {{< /code >}}
 
-{{< code powershell "Windows" >}}
-TODO: Document steps for adding CA root to Windows trust store
+{{< /language-toggle >}}
+
+## Renew self-generated certificates
+
+To keep your Sensu deployment running smoothly, renew your self-generated certificates before they expire.
+Depending on how your certificates are configured, one backend certificate may expire before the others or all three backend certificates may expire at the same time.
+The agent certificate also expires.
+
+This section explains how to find certificate expiration dates, confirm whether certificates have already expired, and renew certificates.
+
+### Find certificate expiration dates
+
+Use this check to find certificate expiration dates so you can renew certificates before they expire and avoid observability interruptions:
+
```

Comment:
I think I need to add something here about changing the cert name in the command to reflect the cert whose expiration they want to check.